.github/workflows/main.yml

```diff
@@ -11,24 +11,32 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
+  build-test:
+    runs-on: ${{ matrix.os }}
+    name: Build & Test (${{ matrix.os }})
     strategy:
+      fail-fast: false
       matrix:
-        compiler: [g++-10, clang++-12]
+        os: [ubuntu-latest, macos-latest]
+        generator: [UnixMakefiles]
+        include:
+          - os: windows-latest
+            generator: VS16Win64
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Install GCC, Clang, CMake
+    - name: Install GCC, CMake
+      if: ${{ matrix.os == 'ubuntu-latest' }}
       run: sudo apt-get install build-essential cmake
 
-    - name: Configure CMake
-      run: mkdir build && cmake -Bbuild -DCMAKE_CXX_COMPILER=${{ matrix.compiler }}
```

Comment:
After this change we no longer build with Clang on Linux. Is that right?

Reply:
I missed that because I was working off an older main branch. I will add to the matrix so we have multiple compilers on Linux.